tasks/config-opsdir/task.sh

```diff
@@ -44,6 +45,38 @@ AZ_CONFIGURATION=$(cat <<-EOF
 }
 EOF
 )
+elif [ ! -z $AZ_2 ];then
+AZ_CONFIGURATION=$(cat <<-EOF
+{
+  "availability_zones": [
+    {
+      "name": "$AZ_1",
+      "cluster": "$AZ_1_CLUSTER_NAME",
+      "resource_pool": "$AZ_1_RP_NAME"
+    },
+    {
+      "name": "$AZ_2",
+      "cluster": "$AZ_2_CLUSTER_NAME",
+      "resource_pool": "$AZ_2_RP_NAME"
+    }
+  ]
+}
+EOF
+)
+else
+AZ_CONFIGURATION=$(cat <<-EOF
+{
+  "availability_zones": [
+    {
+      "name": "$AZ_1",
+      "cluster": "$AZ_1_CLUSTER_NAME",
+      "resource_pool": "$AZ_1_RP_NAME"
+    }
+  ]
+}
+EOF
+)
+fi
```

Comment:
I didn't do a good job explaining the challenge with OPSMAN_IP - turns out I was looking at an old version where you had already switched from OPSMAN_IP to OPSMAN_URI. In this case, the only difference needed would be to use https as the other tasks in the pipeline do rather than http. I will create a new PR just for that instead.